src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transition-property: 300ms;
```

Comment:
The 'transition-property' should specify which CSS properties the transition effect should apply to, such as 'transform'. Consider using 'transition: transform 300ms;' instead to ensure the transition applies correctly.